vignettes/presize.Rmd

```diff
@@ -0,0 +1,189 @@
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+[Bland (2009)](https://www.bmj.com/content/339/bmj.b3985) recommended to
+base study sizes on the width of the confidence interval rather the power of 
+a statistical test. The goal of `presize` is to provide functions for such 
+precision based sample size calculations. For a given sample size, the 
+functions will return the precision (width of the confidence interval), and 
+vice versa.
+
+`presize` is loaded like any other R Package:
+
+```{r setup}
+library(presize)
+```
+
+## Using `presize`
+
+Here we present a couple of examples of using `presize` to determine the suitable sample size for a trial.
+
+### Precision with a fixed N
+
+As preparation for a trial to estimate the sensitivity and specificity of a mobile colposcope, a device for detecting tissue abnormality, a sample size calculation was made to estimate the precision of the sensitivity of the device. A best guess for sensitivity was 75% at a prevalence of 15% with a desired sample size of 250 participants. As such, the question to be answered is how wide is the confidence interval going to be under such a scenario?
+
+We plug the values into the `prec_sens` (precision of sensitivity) function as follows 
+
+```{r}
+(ss <- prec_sens(sens   = .75,  # sensitivity 
+                 prev   = .15,  # prevalence
+                 ntot   = 250,  # sample size
+                 method = "wilson"))
+```
+
+We see that with 250 participants, the confidence interval would be from `r round(ss$lwr, 2) * 100`% (`lwr`) to `r round(ss$upr, 2) * 100`% (`upr`).
+
+Note that Wilson's method of calculating confidence intervals adjusts the point estimate to allow the calculation of a symmetrical CI. This is the `sensadj` variable. For some measures there are multiple methods available. In the case of `prec_sens` (as well as `prec_spec` and `prec_prop`), there are four different approaches for creating confidence intervals, each yielding slightly different results. 
+
+For specificity, we use the `prec_spec` function instead. For demonstration purposes, we can also change the method used to calculate the CI.
+
+```{r}
+prec_spec(spec   = .75,  # specificity 
+          prev   = .15,  # prevalence
+          ntot   = 250,  # sample size
+          method = "exact")
+```
+
+Using the other functions in `presize` is the same. For instance the precision of a mean of 60 with an SD of 10 and 40 observations is calculated as follows.
+
+```{r}
+prec_mean(22, sd = 10, n = 40)
+```
+
+### N for a fixed precision
+
+While fixing the sample size is sometimes necessary, it is perhaps more common to select a sample size based on a given precision. For instance, We want to achieve a sensitivity of a given amount (e.g. plus-minus 5%). This is also possible with `presize`. Using the same code as above, we replace the `ntot` argument with the `conf.width` argument. 
```

Comment:
I would delete the perhaps.
"For instance"...
Capital letter in We not needed.